server/expression.js

```diff
@@ -261,79 +245,6 @@ expression.getTfaProfile_ = function(fileName, geneNames, conditionNames) {
 };
 
 /**
- * Gets the expression matrix data.
- * @param {string} file File name of the expression matrix data.
- * @param {string} exprows Regex selecting the genes.
- * @param {string} expcols Regex selecting the experiment conditions.
- * @return {?expression.Matrix}
- * @private
- */
-expression.getMatrix_ = function(file, exprows, expcols) {
```

Comment:
Removal of this function seems to result in readMatrix_ being never used. You may remove readMatrix_ as well.